tests/basic.rs

```diff
@@ -297,3 +297,12 @@ fn waker() {
     waker.wake();
     r.recv().unwrap();
 }
+
+#[test]
+fn raw() {
+    let (runnable, _handle) = async_task::spawn(async {}, |_| panic!());
+
+    let a = runnable.into_raw();
+    let task = unsafe { Runnable::<()>::from_raw(a) };
+    task.run();
+}
```

Comment:
I would like a more thorough test that passes the task across a simulated FFI boundary. Like, have an `extern "C"` function that takes a pointer, just to see if this works for its intended use case.